internal/driver/fetch_test.go

```diff
@@ -362,6 +362,14 @@ func TestHttpsInsecure(t *testing.T) {
 	if runtime.GOOS == "nacl" {
 		t.Skip("test assumes tcp available")
 	}
+	saveHome := os.Getenv(homeEnv())
```

Comment:
Please add a comment on why this is done here.

Reply:
Done, but as this has been merged already this requires another PR I guess...